timon/db/backends.py

```diff
@@ -69,6 +77,18 @@ async def store_probe_result(self, probename, timestamp, msg, status):
         raise NotImplementedError(
             "Backend store_probe_result func must be implemented")
 
+    async def store_hist_probe_result(self, probename, timestamp, msg, status):
+        """Store an history probe result (state changed)
```

Comment:
store a history  ('h' is pronouned and phonetically not a vowel)

Reply:
done